api/spec/swagger_helper.rb

```diff
@@ -105,6 +106,48 @@
             },
             required: %w[name presentation]
           },
+          order_params: {
+            type: :object,
+            properties: {
+              number: { type: :string },
+              item_total: { type: :number },
+              total: { type: :number },
+              state: { type: :string },
+              adjustment_total: { type: :number },
+              user_id: { type: :string },
+              completed_at: { type: :string, format: :date_time },
+              bill_address_id: { type: :string },
+              ship_address_id: { type: :string },
+              payment_total: { type: :number },
+              shipment_state: { type: :string },
+              payment_state: { type: :string },
+              email: { type: :string, format: :email },
+              special_instructions: { type: :string },
+              created_at: { type: :string, format: :date_time },
+              updated_at: { type: :string, format: :date_time },
+              currency: { type: :string },
+              last_ip_address: { type: :string },
+              created_by_id: { type: :string },
+              shipment_total: { type: :number },
+              additional_tax_total: { type: :number },
+              promo_total: { type: :number },
+              channel: { type: :string },
+              included_tax_total: { type: :number },
+              item_count: { type: :integer },
+              approver_id: { type: :string },
+              approved_at: { type: :string, format: :date_time },
+              confirmation_delivered: { type: :boolean },
+              considered_risky: { type: :boolean },
+              token: { type: :string },
+              canceled_at: { type: :string, format: :date_time },
+              canceler_id: { type: :string },
+              store_id: { type: :string },
```

Comment:
Are we locking an order to the store it was created in? If so I'm thinking we can drop` store_id: { type: :string }` here maybe?

Reply:
@MatthewKennedy so this schema is also used for update action, and we would like to allow app developers later to re-assign the Order to a different Store

Reply:
No problem, all good.